### ISSUE TITLE
Feature Request: Ability to play Catch-up Programs from Channel Guide

### ISSUE THREAD

**Issue by nikhilk0** [xbmc/Official-Kodi-Remote-iOS#681]
Yatse on Android has ability to play Catch-up programs from channel guide. 
If something similar can be implemented on iOS app, that'd be great.

**Comment by wutschel** [comment]
This is new to me. This is done using a add-on for Kodi?

**Comment by nikhilk0** [comment]
Yes. The addon which I'm using supports 7 day catchup. If the program is available as a catchup, I'm able to directly play that from EPG.

It was added in Yatse in version 10.8.0 - https://community.yatse.tv/t/yatse-10-8-0-released/3392
https://community.yatse.tv/t/support-for-catch-up-tv-using-kodi-19/3356

**Comment by wutschel** [comment]
I played with this add-on on my test environment, but I have no idea how this should work from the guide (neither in Kodi itself nor in a yatse installation). Is there a description you can share on how this should behave, and possibly how to configure?

**Comment by nikhilk0** [comment]
Just to be clear, I'm not talking about the 'catch-up tv & more' addon, which is present in default kodi repo. By catch-up, I'm referring to [this](https://github.com/kodi-pvr/pvr.iptvsimple#catchup).

I'm using an addon which generates PVR IPTV playlist and supports catch-up via PVR-IPTV Simple.

In Yatse, I can already aired programs play via EPG as below

![Screenshot_2022-06-12-21-30-27-78_e6837d338de0f94b34f1497044a9a798](https://user-images.githubusercontent.com/45728234/173242180-5e6c35dd-aaad-4345-a5ce-7dcf572407fe.jpg)

**Comment by wutschel** [comment]
Thanks. I now took a quick look at this, and I need to admit this seems more complicated than I thought. For now I will not further dig into this, please don't have too high hopes on this getting implemented.
